core/domain/classifier/algorithm_registry_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, softwar
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for algorithm registry."""
+
+
+import itertools
+import feconf
+
+from core.domain.classifier import algorithm_registry
+from core.domain.classifier.LDAStringClassifier import LDAStringClassifier
+from core.tests import test_utils
+
+
+class AlgorithmRegistryTest(test_utils.GenericTestBase):
+    """Test the classifier algorithm registry class."""
+
+    def test_get_all_classifier_ids(self):
+        expected_algorithm_ids = [
+            'LDAStringClassifier'
+        ]
+        algorithm_ids = algorithm_registry.Registry.get_all_classifier_ids()
+        self.assertItemsEqual(expected_algorithm_ids, algorithm_ids)
+
+    def test_get_all_classifiers(self):
+        num_classifiers = len(list(itertools.chain(*[
+            classifier_class['algorithm_ids']
+            for classifier_class in feconf.ALLOWED_CLASSIFIER_CLASSES
+        ])))
+        classifier_instances = algorithm_registry.Registry.get_all_classifiers()  # pylint: disable=line-too-long
+        self.assertEquals(len(classifier_instances), num_classifiers)
+
+    def test_get_classifier_by_id(self):
+        classifier_id = 'LDAStringClassifier'
+        classifier_instance = algorithm_registry.Registry.get_classifier_by_id(classifier_id)  # pylint: disable=line-too-long
+        self.assertIsInstance(classifier_instance, LDAStringClassifier,
+                              msg='The object is not an instance of '
```

Comment:
Yes, it would indicate to some extent. I was just trying to follow the [pattern](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects_test.py#L422) here.